Streamable/scripts/scr_decklist/scr_decklist.gml

```diff
@@ -0,0 +1,130 @@
+function CardSearch(_name = "", _quantity = 1) constructor
+{
+	name = _name;
+	quantity = _quantity;
+	set = "";
+	collector_number = "";
+	
+	static CreateRequest = function()
+	{
+		var search = "!\"" + name + "\"";
+		var search_type = "name";
+		
+		if set != ""
+		{
+			search = search + " set:\"" + set + "\"";
+			search_type = "search";
+		}
+	
+		if collector_number != ""
+		{
+			search = search + " cn:\"" + collector_number + "\"";
+			search_type = "search";
+		}
+		
+		show_debug_message(search);
+		
+		instance_create_layer(room_width / 2, room_height / 2, "Instances", obj_card_request, { "req": search, "spawn_number": real(quantity), "searchType": search_type })
+	}
+}
+
+function parse_decklist_line(content, index)
+{
+	
+	
+	// Extract the quantity:
+	content = string_trim(content);
+	var str_temp = string_split(content, " ", false, 1);
```

Comment:
this function blows up on empty lines, like if there's a newline at the end of the file or lines between groups of cards. should probably handle it gracefully by skipping empty lines.

it should also be noted that i have **not** verified that all this parsing is the same as the original parsing. would be good for a robust review, but since i already have some suggestions, i plan to go over finer-detail stuff like that after this initial review.

Reply:
Fixed